perf/perf_stress_ng.py

```diff
@@ -130,6 +131,8 @@ def test(self):
         smt = int(re.split(r'=| is ', process.system_output("ppc64_cpu --smt")
                            .decode('utf-8'))[1])
         no_of_cores = int(self.tcpus // smt)
+        clock_freq = int(subprocess.check_output("cat /proc/cpuinfo | grep -m1 clock |sed 's/.*://' | cut -f1 -d'.'", shell=True))
```

Comment:
@kjain101 can we use like below  
```
cmd = "cat /proc/cpuinfo | grep -m1 clock |sed 's/.*://' | cut -f1 -d'.'"
 clock_freq = int(subprocess.check_output(cmd, shell=True)
```

Reply:
Hey naresh yes that looks fine to me. I will do the changes